R/expect_type_linter.R

```diff
@@ -0,0 +1,60 @@
+#' Require usage of expect_type(x, type) over expect_equal(typeof(x), type)
+#'
+#' [testthat::expect_type()] exists specifically for testing the storage type
+#'   of objects. [testthat::expect_equal()], [testthat::expect_identical()], and
+#'   [testthat::expect_true()] can also be used for such tests,
+#'   but it is better to use the tailored function instead.
+#'
+#' @evalRd rd_tags("expect_type_linter")
+#' @seealso [linters] for a complete list of linters available in lintr.
+#' @export
+#' @importFrom glue glue
```

Comment:
That's not necessary if you call `glue::glue()` with its namespace.

Reply:
I added this hoping it would make `roxygen2()` automatically add `glue` to the DESCRIPTION which didn't work. i'll remove it